rockcraft/layers.py

```diff
@@ -251,8 +278,19 @@ def _all_compatible_files(paths: List[Path]) -> bool:
 
     first_file = paths[0]
 
+    permissions_first = [_get_permissions(first_file)]
+
     for other_file in paths[1:]:
-        if paths_collide(str(first_file), str(other_file)):
+        permissions_other = [_get_permissions(other_file)]
+        if paths_collide(
+            str(first_file), str(other_file), permissions_first, permissions_other
+        ):
```

Comment:
it would be useful to have a debug log to print what those files are and their permissions

Reply:
the filenames are already printed by the calling code (in `prune_prime_files()`)
Any suggestions on how to _succinctly_ log the stats of the files?

Reply:
if the files are already printed, that's good enough